src/librustdoc/html/static/main.js

```diff
@@ -1962,6 +1964,21 @@ function defocusSearchBar() {
         }
     };
 
+    /// Render HTML as plaintext:
```

Comment:
Just noticed but that's not a JS comment. :p

Reply:
Yeah, I wrote it like a rustdoc comment, but this *is* JavaScript.

Reply:
It's valid, but unexpected. Please use "//" haha.

Reply:
I converted it to a JSDoc comment.

Reply:
:+1: